web/usuaris/src/components/organisms/UsersTable/UsersTable.tsx

```diff
@@ -38,6 +48,17 @@ export const UsersTable: FC<TUsersTable> = ({ filtersSelected }) => {
 
   const { changeUserStatus } = useUpdateUser()
 
+  const [isModalOpen, setIsModalOpen] = useState<boolean>(false)
+
+  const [idToDelete, setIdToDelete] = useState<string>('')
+
+  const [showDeleteError, setShowDeleteError] = useState<boolean>(false)
+
+  const [isDeletionSuccessful, setIsDeletionSuccessful] =
+    useState<boolean>(false)
+
+  const { isLoading: isDeleteLoading, deleteUserMutation } = useDeleteUser()
```

Comment:
You could return isSuccess in the hook useDeleteUser() too, so you don't need to do the useState for it.